### PR TITLE
V0.1.4 upgrades

### PR DESCRIPTION
**Pull request by sivaprasadreddy**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-      "config:base"
+      "config:base",
+      "schedule:earlyMondays"
     ],
     "includeForks": true,
-    "labels": ["dependencies"],
-    "packageRules": [
-      {
-        "updateTypes": ["minor", "patch", "pin", "digest"],
-        "automerge": true
```
Comment (sivaprasadreddy) [inline renovate.json:10]: @rajadileepkolli As we don't have a concrete test suite to verify the version upgrade validity, I would recommend not to autoMerge for now.

File: generators/constants.js [hunk 2]
```diff
@@ -1,37 +1,49 @@
...
-    POSTGRESQL_IMAGE: 'postgres:15.3-alpine',
-    MARIADB_IMAGE: 'mariadb:11.0',
-    MYSQL_IMAGE: 'mysql:8.0.3',
+    JAVA_VERSION: '17',
+    SPRING_BOOT_VERSION: '3.1.3',
+    SPRING_CLOUD_VERSION: '2022.0.4',
+    SPRING_CLOUD_AWS_VERSION: '3.0.2',
+    SPRING_DEP_MNGMNT_VERSION: '1.1.3',
+    DEFAULT_APP_VERSION: '0.0.1-SNAPSHOT',
+    TEST_CONTAINERS_VERSION: '1.19.0',
```
Comment (rajadilipkolli) [inline generators/constants.js:13]: Should we manage test containers version manually? I prefer using framework supported version

File: generators/constants.js [hunk 3]
```diff
@@ -1,37 +1,49 @@
...
+    SPRINGDOC_OPENAPI_VERSION: '2.1.0',
```
Comment (rajadilipkolli) [inline generators/constants.js:14]: 2.2.0 version is available

File: generators/constants.js [hunk 4]
```diff
@@ -1,37 +1,49 @@
...
+    GRADLE_JACOCO_PLUGIN_VERSION: '0.8.10',
+    GRADLE_SONAR_PLUGIN_VERSION: '4.3.0.3225',
+    GRADLE_OWASP_PLUGIN_VERSION: '8.2.1',
+    GRADLE_BENMANES_VERSIONS_PLUGIN_VERSION: '0.47.0',
+    GRADLE_SPOTLESS_PLUGIN_VERSION: '6.21.0',
+
+    MAVEN_DEPENDENCY_CHECK_PLUGIN_VERSION: '8.3.1',
+    MAVEN_PROPERTIES_PLUGIN_VERSION: '1.2.0',
+    MAVEN_SUREFIRE_PLUGIN_VERSION: '3.1.2',
+    MAVEN_FAILSAFE_PLUGIN_VERSION: '3.1.2',
```
Comment (rajadilipkolli) [inline generators/constants.js:30]: Spring boot also started using 3.1.x so we should use the framework version

**Review by rajadilipkolli — APPROVED** [review]
LGTM, ready to merge

**Pull request merged**